scrapers/Waybig.yml

```diff
@@ -0,0 +1,67 @@
+name: Waybig
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - waybig.com/video/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - waybig.com/pornstars/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@name="twitter:title"]/@content
+      Date:
+        selector: //i[@class="far fa-calendar"]/following-sibling::text()
```

Comment:
```yaml
        selector: //div/i[@class="far fa-calendar"]/../span
```
seems safer

Also a note if you go to the last page by date you will notice some scenes (very few) have weird dates
Using the selector we can get either `0000-00-00` or something like `0000-02-18` 
Regex replacing `^(00.+)` with a `""` still has issues with parseDate so i think we should just ignore those dates